TopReadWidget/WMFTodayTopReadWidgetViewController.swift

```diff
@@ -130,10 +128,39 @@ class WMFTodayTopReadWidgetViewController: UIViewController, NCWidgetProviding {
         perform(#selector(updateView), with: nil, afterDelay: 0.1)
     }
     
+    func updateThemeFromTraitCollection() {
```

Comment:
Any reason for not using super's updateThemeFromTraitCollection? Looks like the same implementation.

Reply:
forgot to delete this 😅